conf.py

```diff
@@ -293,3 +294,8 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'https://docs.python.org/': None}
+
+extlinks = {
+    'oep-status': ('https://openedx.atlassian.net/browse/%s', 'OEP-')
```

Comment:
I'm actually not sure if there's going to be a JIRA board at all. It probably depends on whether I can find a way to make sphinx summarize the status of the OEPs (or have the make file auto-build a page that summarizes the statuses).